state/firewallrules.go

```diff
@@ -26,11 +26,36 @@ import (
 // - juju-controller
 // - juju-application-offer
 type FirewallRule struct {
-	// WellKnownService is the known service for the firewall rules entity.
-	WellKnownService WellKnownServiceType
+	id string
+
+	// WellKnownServiceType is the known service for the firewall rules entity.
+	wellKnownServiceType WellKnownServiceType
+
+	// WellKnownService is the known service for the firewall rules entity as a string.
+	wellKnownService string
```

Comment:
I don't think we want both members when one is a derivation of the the other...

Reply:
:100: